tasks/utils.py

```diff
@@ -50,9 +50,18 @@ def get_multi_python_location(embedded_path=None, rtloader_root=None):
 
     return rtloader_lib, rtloader_headers, rtloader_common_headers
 
+def has_both_python(python_runtimes):
+    python_runtimes = python_runtimes.split(',')
+    return '2' in python_runtimes and '3' in python_runtimes
+
+def get_win_py_runtime_var(python_runtimes):
+    python_runtimes = python_runtimes.split(',')
+
+    return "PY2_RUNTIME" if '2' in python_runtimes else "PY3_RUNTIME"
```

Comment:
Probably a matter of discussion for another PR, but this magic string variable thing is a little nasty.